MSAL/src/configuration/MSALCacheConfig.m

```diff
@@ -122,7 +125,10 @@ - (NSArray *)createTrustedApplicationListFromPaths:(NSArray<NSString *> *)appPat
     for (NSString *appPath in appPaths)
     {
         SecTrustedApplicationRef app = nil;
-        status = SecTrustedApplicationCreateFromPath([appPath UTF8String], &app);
+#pragma clang diagnostic push
+    #pragma clang diagnostic ignored "-Wdeprecated-declarations"
+         status = SecTrustedApplicationCreateFromPath([appPath UTF8String], &app);
```

Comment:
Is it safe to ignore deprecation warning? Can we replace this with an alternative?

Reply:
+1